lib/Sema/CSDiagnostics.cpp

```diff
@@ -9576,3 +9576,12 @@ bool IncorrectSlabLiteralCount::diagnoseAsError() {
   emitDiagnostic(diag::slab_literal_incorrect_count, lhsCount, rhsCount);
   return true;
 }
+
+bool WarnAboutExistentialOpenedForCallArgumentUntilFutureRelease::
+    diagnoseAsError() {
+  auto overload = *getCalleeOverloadChoiceIfAvailable(getLocator());
```

Comment:
Nit: we have `getOverloadChoice` for this which force unwraps inside.

Reply:
This also looks through to the callee locator, though.

Reply:
Oh, I missed that. Let's add a variant without `IfAvailable` to solution just like `getOverloadChoice`.

Reply:
Without a forwarding method on `FailureDiagnostic`, right?